modules/nodes-nodes-audit-log-basic.adoc

```diff
@@ -20,13 +20,13 @@ Audit works at the API server level, logging all requests coming to the server.
 |`impersonatedUser` |Optional. The impersonated user information, if the request is impersonating another user.
 |`sourceIPs` |Optional. The source IPs, from where the request originated and any intermediate proxies.
 |`userAgent` |Optional. The user agent string reported by the client. Note that the user agent is provided by the client, and must not be trusted.
-|`objectRef` |Optional. The object reference this request is targeted at. This does not apply for List-type requests, or non-resource requests.
-|`responseStatus` |Optional. The response status, populated even when the ResponseObject is not a Status type. For successful responses, this will only include the Code. For non-status type error responses, this will be auto-populated with the error Message.
-|`requestObject` |Optional. The API object from the request, in JSON format. The RequestObject is recorded as is in the request (possibly re-encoded as JSON), prior to version conversion, defaulting, admission or merging. It is an external versioned object type, and might not be a valid object on its own. This is omitted for non-resource requests and is only logged at Request Level and higher.
-|`responseObject` |Optional. The API object returned in the response, in JSON format. The ResponseObject is recorded after conversion to the external type, and serialized as JSON. This is omitted for non-resource requests and is only logged at Response Level.
+|`objectRef` |Optional. The object reference this request is targeted at. This does not apply for `List`-type requests, or non-resource requests.
+|`responseStatus` |Optional. The response status, populated even when the `ResponseObject` is not a `Status` type. For successful responses, this will only include the code. For non-status type error responses, this will be auto-populated with the error message.
+|`requestObject` |Optional. The API object from the request, in JSON format. The `RequestObject` is recorded as is in the request (possibly re-encoded as JSON), prior to version conversion, defaulting, admission or merging. It is an external versioned object type, and might not be a valid object on its own. This is omitted for non-resource requests and is only logged at request level and higher.
+|`responseObject` |Optional. The API object returned in the response, in JSON format. The `ResponseObject` is recorded after conversion to the external type, and serialized as JSON. This is omitted for non-resource requests and is only logged at response level.
```

Comment:
Is the difference in capitalization correct here between the field column and the description for 24, 25, 26?
'requestObect` in the field column, `RequestObject` in the description
'responseObect` in the field column, `ResponseObject` in the description

Reply:
Yes it is. ResponseObject is the object, but the field that shows in the logs is responseObject. Thanks for checking!